src/bin/inx-chronicle/api/stardust/core/routes.rs

```diff
@@ -250,19 +253,24 @@ fn create_output_metadata_response(metadata: OutputMetadataResult) -> OutputMeta
             .map(|spent_md| spent_md.transaction_id.to_hex()),
         milestone_index_booked: *metadata.booked.milestone_index,
         milestone_timestamp_booked: *metadata.booked.milestone_timestamp,
-        ledger_index: metadata.ledger_index.0,
+        ledger_index: ledger_index.0,
```

Comment:
I'd use the `metadata` here. If for some reason, something is wrong with the `ledger_index` it will be a lot harder to debug.

Reply:
It's literally the same value. There's no reason to push it through the db call to to pull it back.